examples/webserver/hunchentoot.clj

```diff
@@ -0,0 +1,48 @@
+(ns webserver.hunchentoot
+  (:require [abclj.quicklisp :refer [quickload]]
+            [abclj.core :refer [defun with-cl]]))
+
+(quickload "hunchentoot")
+(quickload "cl-json")
+(quickload "alexandria")
+
+(defun server ()
+  (defvar *acceptor*)
```

Comment:
Defvar seems misplaced here.  Shoudn't it be toplevel, as always?

Reply:
defvar in this case is CL code, and not clojure one. There is no defvar defined in abclj, the closest one is `setvar` which assign a value to a symbol (if the symbol didn't exist it is interned on the CL package).

Reply:
On this https://github.com/lsevero/abclj/pull/8 I tried to implement a toplevel `defvar` but there might by a huge syncronization problem trying to keep the value in CL environment to be the same as the one defined in CLJ. For now, you can use `setvar` which if closer in behavior to `defparameter` or use `with-cl` like this:

```clj
(with-cl
  '(defvar *alive* 1))
```